### PR TITLE
Update CSM for Resiliency documentation update

### PR DESCRIPTION
**Pull request by alikdell**
# Description
Update CSM for Resiliency documentation update for
Release note
Multi-array support and connectivity state on multi-array setup

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| #https://github.com/dell/csm/issues/333 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

**Review by rbo54 — APPROVED** [review]

File: content/docs/resiliency/_index.md [hunk 1]
```diff
@@ -106,11 +106,11 @@ The following provisioning types are supported and have been tested:
 * Use of the above volumes with Pods created by StatefulSets.
 * Up to 12 or so protected pods on a given node.
 * Failing up to 3 nodes at a time in 9 worker node clusters, or failing 1 node at a time in smaller clusters. Application recovery times are dependent on the number of pods that need to be moved as a result of the failure. See the section on "Testing and Performance" for some of the details.
+* Multi-array are supported. Incase of CSI Driver PowerSale and CSI Driver Unity, if any one of the array is not connected, the array connectivity will be false.
```
Comment (rbo54) [inline content/docs/resiliency/_index.md:109]: PowerScale, not PowerSale, and please mention that for PowerFlex the array connectivity will be determined by connection to the default array.

**Review by rbo54 — CHANGES_REQUESTED** [review]

**Review by rbo54 — APPROVED** [review]

**Review by shaynafinocchiaro — DISMISSED** [review]

File: content/docs/resiliency/release/_index.md [hunk 2]
```diff
@@ -13,8 +13,10 @@ Description: >
 
 ### New Features/Changes
 
+- Resiliency protection on driver node pods, see [CSI node failure protection](https://github.com/dell/csm/issues/145).
```
Comment (randeepsharma) [inline content/docs/resiliency/release/_index.md:16]: The below sentences  need to move from above to here.

- Added support for PowerScale CSI Driver.
- Support for node taint when driver pod is unhealthy

Comment (randeepsharma) [inline content/docs/resiliency/release/_index.md:16]: The below sentences  need to move from above to here.

- Added support for PowerScale CSI Driver.
- Support for node taint when driver pod is unhealthy

Comment (alikdell) [reply content/docs/resiliency/release/_index.md:16]: I am not sure those are redundant information for sure, I will add the feature link of PowerScale in new feature section

File: content/docs/resiliency/_index.md [hunk 3]
```diff
@@ -106,11 +106,11 @@ The following provisioning types are supported and have been tested:
 * Use of the above volumes with Pods created by StatefulSets.
 * Up to 12 or so protected pods on a given node.
 * Failing up to 3 nodes at a time in 9 worker node clusters, or failing 1 node at a time in smaller clusters. Application recovery times are dependent on the number of pods that need to be moved as a result of the failure. See the section on "Testing and Performance" for some of the details.
+* Multi-array are supported. In case of CSI Driver PowerScale and CSI Driver Unity, if any one of the array is not connected, the array connectivity will be false. For CSI Driver Powerflex connectivity will be determined by connection to default array.
```
Comment (hoppea2) [inline content/docs/resiliency/_index.md:109]: CSI Driver *for* PowerScale / CSI Driver *for* Unity / CSI Driver *for* PowerFlex

Comment (alikdell) [reply content/docs/resiliency/_index.md]: updated

**Review by shaynafinocchiaro — DISMISSED** [review]

**Review by hoppea2 — DISMISSED** [review]

File: content/docs/resiliency/release/_index.md [hunk 4]
```diff
@@ -8,13 +8,16 @@ Description: >
 
 ## Release Notes - CSM Resiliency 1.2.0
 
-- Added support for PowerScale CSI Driver.
+- Support for CSI Driver for PowerScale.
```
Comment (coulof) [inline content/docs/resiliency/release/_index.md:11]: Can you update his table too : https://github.com/dell/csm-docs/blob/main/content/docs/_index.md#csm-modules-support-matrix-for-dell-csi-drivers ?
thx

File: content/docs/resiliency/release/_index.md (hunk 2 above)
Comment (shanmydell) [reply content/docs/resiliency/release/_index.md:16]: @alikdell : Dont need to duplicate , pls move it here

**Review by shanmydell — CHANGES_REQUESTED** [review]
Please address review comments from @coulof @randeepsharma

File: content/docs/resiliency/release/_index.md (hunk 4 above)
Comment (randeepsharma) [reply content/docs/resiliency/release/_index.md:11]: @coulof This is already addressed. Please look at release-1.3 branch.
https://github.com/dell/csm-docs/blob/release-1.3/content/docs/_index.md

**Review by randeepsharma — APPROVED** [review]

**Review by hoppea2 — APPROVED** [review]

**Pull request merged**